.github/workflows/bump-version.yaml

```diff
@@ -0,0 +1,93 @@
+# DO NOT EDIT THIS FILE.
+# Instead, edit the jsonpatch file (actually YAML) in .github/bump-version-patch.yaml
+# For docs, see github-actions in the IronCoreLabs/depot repo.
+
+name: Bump Version
+'on':
+  push:
+    branches:
+    - main
```

Comment:
As written, this will make a new release every time a PR is merged to `main`, even if it's just a PR like this one that modifies the workflows and has no code changes. Consider the suggestions in https://github.com/IronCoreLabs/depot/issues/302; those ideas haven't been discussed and fully accepted yet, which is why they're not codified in the template workflows in `depot`.

As an alternative that requires less work, you could just delete these 3 lines (using a patch to delete `/on/push`) and run `bump-version` manually when you want to make a release.

Reply:
I guess I'm incorrect in my understanding of what bump-version does. I always thought it was just the workflow that ups the version number in the package.json to the next `pre` version. But I do see here that it also does push tags for a release. What's the reasoning for that?

Reply:
https://github.com/IronCoreLabs/depot/tree/main/github-actions#continuous-deployment has the docs for those workflows. Bump-version starts the chain of workflows, because making a release has to start with version management.